.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,71 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
```

Comment:
Interesting, I didn't know about CodeQL. Is there a preview of what kind of things this would report? I'm only concerned if it would add noise like it's often the case with `npm audit` and Dependabot (see e.g. [npm audit: Broken by Design](https://overreacted.io/npm-audit-broken-by-design/)).

If it provides helpful suggestions then I'm open. I'm wondering though if it has to run in a cron job? Since it analyses the code, doesn't it make sense to run it when code changes?

Reply:
Honestly I don't know too much as well ^^

I just threw all these scanners because my team is complaining about "vulnerabilities". I'm not sure if they don't raise false positives like the examples given in that blog post (Dan the man!)

Anyways it looks like this, and so far only the generated files are getting flagged. I added a config file to ignore `dist` so nothing should show up in the main(this) repository.

![image](https://user-images.githubusercontent.com/9292261/152293801-d15966f3-6ef3-416e-988b-14c6e92d8dbb.png)

Reply:
Ok, thanks for the additional info then! 🙂

Well if there's no strong reason to introduce this currently, then I'd wait with the code scanners.